src/ra.hrl

```diff
@@ -44,12 +44,18 @@
                           suspended |
                           disconnected.
 
+-type ra_voter_status() :: yes | {no, ra_nonvoter_reason()}.
```

Comment:
should it be `voter` / `non_voter` instead of `yes` / `no`?

Reply:
I'm trying to avoid stuttering a-la "voter => voter". And naming is hard ofc.

How about then changing the map key as well? Any suggestions?